### PR TITLE
Bugfix 36

### LINKED ISSUE #36: remove deprecated views
Some views (and their associated functions) appear to be deprecated. One example is HomeHelp in client (form is greyed out, view functions exist but refer to form), but there may be others.

### PR DESCRIPTION
**Pull request by itay-sho**
Resolves #36

### CODE REVIEW

File: server/templates/registration/login.html [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-      <div class="form-group">
-        <label class="control-label  ">סיסמא</label>
-        {{ form.password }}
-      </div>
-
-    <input type="submit" value="התחבר" />
-    <input type="hidden" name="next" value="{{ next }}" />
-  </form>
-
-{% endblock %}
```
Comment (stavlocker) [inline server/templates/registration/login.html:34]: Why'd you delete this file? It seems necessary to me.
When I try to run in your branch and enter /server it tries to redirect me to `http://localhost:8000/accounts/login/?next=/server/` but then throws a `TemplateDoesNotExist at /accounts/login/` error.

File: server/templates/registration/logged_out.html [hunk 2]
```diff
@@ -1,6 +0,0 @@
-{% extends "server/template.html" %}
-
-{% block body %}
-  <p>התנתקת בהצלחה!</p>
-  <a href="{% url 'login'%}">לחץ כאן להתחבר שוב.</a>
-{% endblock %}
```
Comment (stavlocker) [inline server/templates/registration/logged_out.html:6]: Again, isn't this necessary (look at comment from login.html)

File: server/urls.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 urlpatterns = [
     path('', views.index, name="index"),
-    path('volunteer_table', views.show_all_volunteers, name='show_all_volunteers'),
+    path('volunteer_table/', views.show_all_volunteers, name='show_all_volunteers'),
```
Comment (stavlocker) [inline server/urls.py:7]: Add a trailing slash to help_table too

File: server/templates/registration/login.html (hunk 1 above)
Comment (itay-sho) [reply server/templates/registration/login.html]: yup

File: server/templates/registration/logged_out.html (hunk 2 above)
Comment (itay-sho) [reply server/templates/registration/logged_out.html]: fixed

File: server/urls.py (hunk 3 above)
Comment (itay-sho) [reply server/urls.py:7]: just did. can you check if this is what you meant?

**Comment by itay-sho** [comment]
back to you

**Pull request merged**